src/propagation.cpp

```diff
@@ -149,18 +145,11 @@ std::vector<ot::string_view> getPropagationHeaderNames(const std::set<Propagatio
       headers.push_back(propagation_headers[style].sampling_priority_header);
```

Comment:
(Above, in the signature of `getPropagationHeaderNames`): maybe remove the `bool` if it's not needed anymore.